.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,41 @@
+name: Deploy Hugo Docs GitHub Pages
+
+on:
+  push:
+    branches:
+      - auto-deploy-docs-to-gh-pages  # Deploy on pushing to the default branch
+  pull_request:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true  # Fetch Hugo themes (true OR recursive)
+          fetch-depth: 0    # Fetch all history for .GitInfo and .Lastmod
+
+      - name: Setup Hugo
+        uses: peaceiris/actions-hugo@v2
+        with:
+          hugo-version: '0.76.5'
+          extended: false
+
+      - name: Build with Hugo
+        run: cd ./docs && hugo -D -F
+
+      - name: Replace http:// to https:// in HTML files # Conform to the Enforce HTTPS rule of GitHub Pages
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16.10.0'
+      - run: cd ./docs && node enforce-https-rule-gh-pages.js

```

Comment:
```suggestion
      - run: node enforce-https-rule-gh-pages.js
         working-directory: ./docs/
```

Reply:
Entirely removed this step and the related script.